systemd/README.md

```diff
@@ -0,0 +1,207 @@
+
+[Install]
+WantedBy=default.target
+```
+
+Update the `ExecStart` line to match your configuration. To use offline mode rather than the rinkeby testnet, replace the `ExecStart` line with the following:
+
+```
+ExecStart=/home/livepeer/go-livepeer/livepeer -network offchain -orchestrator -orchSecret osecret.txt -serviceAddr 127.0.0.1:8935 -orchAddr 0.0.0.0:8935
+```
+
+* Save the document and exit `vi` by first hitting the `ESC` key, then `:` followed by `wq` and `<ENTER>`
+
+* Enable and start the services:
+
+```bash
+systemctl enable orchestrator.service
+systemctl start orchestrator.service
+```
+
+* To monitor the service:
+
+```bash
+journalctl -f --unit=orchestrator.service
+```
+
+* Create the `transcoder.service` file:
+
+```bash
+vi /etc/systemd/system/transcoder.service
+```
+
+* Push the `i` key to get into insertion mode.
+
+* Paste the following:
+
+```
+[Unit]
+Description=go-livepeer transcoder daemon service
+After=network.target
+
+[Service]
+User=livepeer
+Group=livepeer
+Type=simple
+Restart=always
+RestartSec=90s
+WorkingDirectory=/home/livepeer/go-livepeer/
+ExecStart=/home/livepeer/go-livepeer/livepeer -network rinkeby -transcoder -orchAddr 127.0.0.1:8935 -orchSecret osecret.txt -nvidia 0
+
+[Install]
+WantedBy=default.target
+```
+
+Update the `ExecStart` line to match your configuration. To use offline mode rather than the rinkeby testnet, replace the `ExecStart` line with the following:
+
+```
+ExecStart=/home/livepeer/go-livepeer/livepeer -network offchain -transcoder -orchAddr 127.0.0.1:8935 -orchSecret osecret.txt -nvidia 0
+```
+
+* Save the document and exit `vi` by first hitting the `ESC` key, then `:` followed by `wq` and `<ENTER>`
+
+* Enable and start the services:
+
+```bash
+systemctl enable transcoder.service
+systemctl start transcoder.service
+```
+
+* To monitor the service:
+
+```bash
+journalctl -f --unit=transcoder.service
+```
+
+To run the orchestrator and transcoder as a single combined service:
+
+* Create the `orchtran.service` file:
+
+```bash
+vi /etc/systemd/system/orchtran.service
+```
+
+* Push the `i` key to get into insertion mode.
+
+* Paste the following:
+
+```
+[Unit]
+Description=go-livepeer orchestrator+transcoder daemon service
+After=network.target
+
+[Service]
+User=livepeer
+Group=livepeer
+Type=simple
+Restart=always
+RestartSec=90s
+WorkingDirectory=/home/livepeer/go-livepeer/
+ExecStart=/home/livepeer/go-livepeer/livepeer -network rinkeby -orchestrator -transcoder -pricePerUnit 1 -nvidia 0 -initializeRound=true -serviceAddr=127.0.0.1:8935
+
+[Install]
+WantedBy=default.target
+```
+
+Update the `ExecStart` line to match your configuration. To use offline mode rather than the rinkeby testnet, replace the `ExecStart` line with the following:
+
+```
+ExecStart=/home/livepeer/go-livepeer/livepeer -network offchain -orchestrator -transcoder -serviceAddr 127.0.0.1:8935 -orchAddr 0.0.0.0:8935 -nvidia 0
```

Comment:
when running in `OT` mode we don't need to supply `-orchAddr` (see comment above).

Reply:
thank again :-)  - removed in 53e33331df01e4a4e81376ba82c5284c5b3fe0e0